Assets/HoloToolkit/Input/Scripts/FocusedObjectMessageSender.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+using HoloToolkit.Unity;
+
+public class FocusedObjectMessageSender : MonoBehaviour
```

Comment:
My vote: Only the change to expose FocusedObject added to the HoloToolkit-Unity.
My suggestion: Use the class you wrote for your application.

=)